analysis/demonhunter/src/shadowlands/conduits/FelDefender.tsx

```diff
@@ -0,0 +1,60 @@
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import ConduitSpellText from 'parser/ui/ConduitSpellText';
+import SPELLS from 'common/SPELLS';
+import React from 'react';
+import Events, { CastEvent } from 'parser/core/Events';
+import { FEL_DEFENDER_COOLDOWN_REDUCTION } from '@wowanalyzer/demonhunter'
+import { SpellLink } from 'interface';
+
+/**
+ * Example log:
+ * https://www.warcraftlogs.com/reports/kvaBpJcMdqG2FKXC#fight=1&type=damage-done&source=4 Rank 7
+ * https://www.warcraftlogs.com/reports/bwk8aGzgdcYMKfCF#fight=1&type=damage-done&source=17 Rank 6
+ */
+
+class FelDefender extends Analyzer {
+
+  conduitRank: number = 0;
+  blurCount: number = 0;
+
```

Comment:
Add a variable something like:
`CDR_ABILITY = this.selectedCombatant.spec === SPECS.HAVOC_DEMON_HUNTER ? SPELLS.BLUR: SPELLS.FIERY_BRAND;`